src/cmd-init

```diff
@@ -100,20 +113,31 @@ mkdir -p src
      fi
  fi)
 
+
+installer_bn=$(basename ${INSTALLER})
+checksums_bn=$(basename ${INSTALLER_CHECKSUM})
 mkdir -p installer
+
+if [ -n "${INSTALLER_DIR}" ] && [ "${FORCE}" != "1" ]; then
+    if (cd "${INSTALLER_DIR}" && sha256sum -c "${checksums_bn}"); then
+        (cd installer
+         cp --reflink=auto "${INSTALLER_DIR}"/"${installer_bn}" .
+         cp --reflink=auto "${INSTALLER_DIR}"/"${checksums_bn}" .
```

Comment:
Though why even copy it? We could just `ln -s` here, right? There's precedence at least doing this with how we handle local paths to checkouts of `fedora-coreos-config`.

Reply:
I think really we want `ln` and then fall back to `cp --reflink=auto`, like what ostree does.

But in the end we're going to be dropping Anaconda hopefully soon anyways...